### PR TITLE
added unix60, moved togehter with southpaw75 into fr4 folder

### PR DESCRIPTION
**Pull request by mkdl**
## Description

Added the "Unix60" keyboard. Moved the "Unix60" together with the "southpaw75" into the new folder "fr4". fr4 is a team effort by the kelvin hall and me. We worked on both keyboards together and will continue to do so in the future under the name fr4.

## Types of Changes

- [x ] Keyboard addition 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/fr4/unix60/config.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define CONFIG_H
```
Comment (drashna) [inline keyboards/fr4/unix60/config.h:19]:
```suggestion
```

File: keyboards/fr4/unix60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "unix60.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```
Comment (drashna) [inline keyboards/fr4/unix60/keymaps/default/keymap.c:21]:
```suggestion
	LAYOUT(
```

File: keyboards/fr4/unix60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8,
+		KC_9, KC_0, KC_MINS, KC_EQL, KC_B, KC_BSPC, KC_C, KC_TAB, KC_Q,
+		KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P,
+		KC_LBRC, KC_RBRC, KC_BSLS, KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G,
+		KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT, KC_LSFT,
+		KC_NUBS, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM,
+		KC_DOT, KC_SLSH, KC_RSFT, KC_A, KC_LGUI, KC_LALT, KC_SPC, KC_RALT, MO(1)),
+
+	KEYMAP(
```
Comment (drashna) [inline keyboards/fr4/unix60/keymaps/default/keymap.c:30]:
```suggestion
	LAYOUT(
```

File: keyboards/fr4/unix60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= no    # Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no
+RGBLIGHT_ENABLE ?= no
```
Comment (drashna) [inline keyboards/fr4/unix60/rules.mk:56]:
```suggestion
# MCU name
MCU = atmega32u4

# Bootloader 
BOOTLOADER = caterina

# Build Options
#   comment out to disable the options.
#
BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes	# Mouse keys
EXTRAKEY_ENABLE = yes	# Audio control and System control
CONSOLE_ENABLE = no	# Console for debug
COMMAND_ENABLE = no    # Commands for debug and configuration
SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
AUDIO_ENABLE = no
RGBLIGHT_ENABLE = no
```

File: keyboards/fr4/unix60/unix60.h [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+
+#include "quantum.h"
+
+#define KEYMAP( \
```
Comment (drashna) [inline keyboards/fr4/unix60/unix60.h:22]:
```suggestion
#define LAYOUT( \
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Could you retarget this to `develop`, since it's changing the folder location (and therefore compile command) for the keyboard. 

You can edit the title, and select "develop" instead of "master".

File: keyboards/fr4/unix60/readme.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Unix60
+
+![unix60](https://raw.githubusercontent.com/mkdl/Unix60/main/Pictures/Top.JPG)
```
Comment (fauxpark) [inline keyboards/fr4/unix60/readme.md:3]: Please rehost this on imgur, and add `l` to the end of the filename to make it thumbnail sized.

File: keyboards/fr4/unix60/readme.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+HHKB layout FR4 Board
+
+* Keyboard Maintainer: [mkdl](https://github.com/mkdl/)
+* Hardware Supported: Unix60 Pro Micro PCB
+* Hardware Availability: https://github.com/mkdl/unix60
+
+Make example for this keyboard (after setting up your build environment):
+
+    make unix60:default
+
```
Comment (fauxpark) [inline keyboards/fr4/unix60/readme.md:14]:
```suggestion
```

File: keyboards/fr4/unix60/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "unix60.h"
```
Comment (fauxpark) [inline keyboards/fr4/unix60/keymaps/default/keymap.c:17]:
```suggestion

#include QMK_KEYBOARD_H
```

File: keyboards/fr4/unix60/rules.mk [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no	# Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/fr4/unix60/rules.mk:19]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = full     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by mkdl** [comment]
> Could you retarget this to `develop`, since it's changing the folder location (and therefore compile command) for the keyboard.
> 
> You can edit the title, and select "develop" instead of "master".

done

File: keyboards/fr4/unix60/readme.md (hunk 6 above)
Comment (mkdl) [reply keyboards/fr4/unix60/readme.md]: done

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/fr4/unix60/readme.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# Unix60
+
+![unix60](https://i.imgur.com/ldvKr90.jpgl)
```
Comment (fauxpark) [inline keyboards/fr4/unix60/readme.md:3]:
```suggestion
![unix60](https://i.imgur.com/ldvKr90l.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Make example in the readme for the southpaw75 needs to be updated as well.

**Review by drashna — APPROVED** [review]

File: keyboards/fr4/unix60/unix60.h [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, K05, K06, K07, K08, K10, K11, K12, K13, K14,       \
+    K15, K16, K17, K18, K20, K21, K22, K23, K24, K25, K26, K27, K28, K30, K31,  \
+    K32, K33, K34, K35, K36, K37, K38, K40, K41, K42, K43, K44, K45, K46, K47,  \
+    K48, K50, K51, K52, K53, K54, K55, K56, K57, K58, K60, K61, K62, K63,       \
+              K64, K65,         K66,                    K67, K68                \
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/unix60.h:27]:
```suggestion
#define LAYOUT_all( \
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K10, K11, K12, K13, K14, K15, K16, \
    K17, K18, K20, K21, K22, K23, K24, K25, K26, K27, K28, K30, K31, K32,           \
    K33, K34, K35, K36, K37, K38, K40, K41, K42, K43, K44, K45, K46, K47,           \
    K48, K50, K51, K52, K53, K54, K55, K56, K57, K58, K60, K61, K62, K63,           \
         K64, K65,                K66,                     K67, K68                 \
```

File: keyboards/fr4/unix60/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+		KC_DOT, KC_SLSH, KC_RSFT, KC_A, KC_LGUI, KC_LALT, KC_SPC, KC_RALT, MO(1)),
+
+	LAYOUT(
+		KC_TILD, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8,
+		KC_F9, KC_F10, KC_F10, KC_F11, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/keymaps/default/keymap.c:38]:
```suggestion
	LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_B,    KC_BSPC, KC_C,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_A,
                 KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, MO(1)
    ),

	LAYOUT_all(
        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F10,  KC_F11,  KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_UP,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
                 KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS
    ),
```

File: keyboards/fr4/unix60/info.json [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
... "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":10, "y":0}, {"x":11, "y":0}, {"x":12, "y":0}, {"x":13, "y":0, "w":2}, {"x":15.25, "y":0}, {"x":16.25, "y":0}, {"x":0, "y":1, "w":1.5}, {"x":1.5, "y":1}, {"x":2.5, "y":1}, {"x":3.5, "y":1}, {"x":4.5, "y":1}, {"x":5.5, "y":1}, {"x":6.5, "y":1}, {"x":7.5, "y":1}, {"x":8.5, "y":1}, {"x":9.5, "y":1}, {"x":10.5, "y":1}, {"x":11.5, "y":1}, {"x":12.5, "y":1}, {"x":13.5, "y":1, "w":1.5}, {"x":0, "y":2, "w":1.75}, {"x":1.75, "y":2}, {"x":2.75, "y":2}, {"x":3.75, "y":2}, {"x":4.75, "y":2}, {"x":5.75, "y":2}, {"x":6.75, "y":2}, {"x":7.75, "y":2}, {"x":8.75, "y":2}, {"x":9.75, "y":2}, {"x":10.75, "y":2}, {"x":11.75, "y":2}, {"x":12.75, "y":2, "w":2.25}, {"x":15.25, "y":2}, {"x":0, "y":3, "w":1.25}, {"x":1.25, "y":3}, {"x":2.25, "y":3}, {"x":3.25, "y":3}, {"x":4.25, "y":3}, {"x":5.25, "y":3}, {"x":6.25, "y":3}, {"x":7.25, "y":3}, {"x":8.25, "y":3}, {"x":9.25, "y":3}, {"x":10.25, "y":3}, {"x":11.25, "y":3}, {"x":12.25, "y":3, "w":1.75}, {"x":14, "y":3}, {"x":1.75, "y":4}, {"x":2.75, "y":4, "w":1.5}, {"x":4.25, "y":4, "w":7}, {"x":11.25, "y":4, "w":1.5}, {"x":12.75, "y":4}]
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/info.json:9]:
```suggestion
        "LAYOUT_all": {
            "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":10, "y":0}, {"x":11, "y":0}, {"x":12, "y":0}, {"x":13, "y":0, "w":0.6}, {"x":13.6, "y":0, "w":0.8}, {"x":14.4, "y":0, "w":0.6}, {"x":0, "y":1, "w":1.5}, {"x":1.5, "y":1}, {"x":2.5, "y":1}, {"x":3.5, "y":1}, {"x":4.5, "y":1}, {"x":5.5, "y":1}, {"x":6.5, "y":1}, {"x":7.5, "y":1}, {"x":8.5, "y":1}, {"x":9.5, "y":1}, {"x":10.5, "y":1}, {"x":11.5, "y":1}, {"x":12.5, "y":1}, {"x":13.5, "y":1, "w":1.5}, {"x":0, "y":2, "w":1.75}, {"x":1.75, "y":2}, {"x":2.75, "y":2}, {"x":3.75, "y":2}, {"x":4.75, "y":2}, {"x":5.75, "y":2}, {"x":6.75, "y":2}, {"x":7.75, "y":2}, {"x":8.75, "y":2}, {"x":9.75, "y":2}, {"x":10.75, "y":2}, {"x":11.75, "y":2}, {"x":12.75, "y":2}, {"x":13.75, "y":2, "w":1.25}, {"x":0, "y":3, "w":1.25}, {"x":1.25, "y":3}, {"x":2.25, "y":3}, {"x":3.25, "y":3}, {"x":4.25, "y":3}, {"x":5.25, "y":3}, {"x":6.25, "y":3}, {"x":7.25, "y":3}, {"x":8.25, "y":3}, {"x":9.25, "y":3}, {"x":10.25, "y":3}, {"x":11.25, "y":3}, {"x":12.25, "y":3, "w":1.75}, {"x":14, "y":3}, {"x":1.5, "y":4}, {"x":2.5, "y":4, "w":1.5}, {"x":4, "y":4, "w":7}, {"x":11, "y":4, "w":1.5}, {"x":12.5, "y":4}]
```

File: keyboards/fr4/unix60/info.json [hunk 14]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "unix60", 
+    "url": "https://github.com/mkdl/Unix60", 
+    "maintainer": "mkdl", 
+    "width": 17.25,
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/info.json:5]:
```suggestion
    "width": 15, 
```

File: keyboards/fr4/unix60/config.h [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/config.h:24]:
```suggestion
#define VENDOR_ID       0x4652 // "FR"
#define PRODUCT_ID      0x5558 // "UX"
```

File: keyboards/fr4/unix60/config.h [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    mkdl
```
Comment (noroadsleft) [inline keyboards/fr4/unix60/config.h:26]:
```suggestion
#define MANUFACTURER    FR4Boards
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/11195/files).

**Review by noroadsleft — APPROVED** [review]

**Review by mkdl — COMMENTED** [review]
ok

**Review by mkdl — COMMENTED** [review]
I thought it got approved already, but now the suggestions are "outdated and cannot be applied" Please help :)

File: keyboards/fr4/unix60/info.json (hunk 13 above)
Comment (mkdl) [reply keyboards/fr4/unix60/info.json]: Please resuggest. Its outdated and I cant commit it. Thanks

File: keyboards/fr4/unix60/unix60.h (hunk 11 above)
Comment (mkdl) [reply keyboards/fr4/unix60/unix60.h]: Please resuggest. Its outdated and I cant commit it. Thanks

**Review by fauxpark — APPROVED** [review]

**Pull request merged**